packages/charts/react-charting/src/components/DonutChart/DonutChart.base.tsx

```diff
@@ -324,16 +343,23 @@ export class DonutChartBase extends React.Component<IDonutChartProps, IDonutChar
   };
 
   private _valueInsideDonut(valueInsideDonut: string | number | undefined, data: IChartDataPoint[]) {
-    const highlightedLegend = this._getHighlightedLegend();
-    if (valueInsideDonut !== undefined && (highlightedLegend !== '' || this.state.showHover)) {
-      let legendValue = valueInsideDonut;
-      data!.map((point: IChartDataPoint, index: number) => {
-        if (point.legend === highlightedLegend || (this.state.showHover && point.legend === this.state.legend)) {
-          legendValue = point.yAxisCalloutData ? point.yAxisCalloutData : point.data!;
+    const highlightedLegends = this._getHighlightedLegend().filter(legend => legend !== undefined);
+
+    if (this.state.showHover) {
+      const legendValue = data.find(point => point.legend === this.state.legend);
```

Comment:
>legendValue [](http://example.com/codeflow?start=12&length=11)

call this variable point

Reply:
calling it pointValue as point is used in data.find